.gitignore

```diff
@@ -7,3 +7,5 @@ Propel/om/*
 Propel/map/*
 
 vendor/*
+_.idea
+.idea
```

Comment:
This should be reverted as it's individual to your dev environment (you can ignore them locally).